services/app-web/src/localAuth/LocalLogin.tsx

```diff
@@ -46,12 +47,20 @@ const localUsers: LocalUserType[] = [
         role: 'CMS_USER',
         stateAssignments: [],
     },
+    {
+        id: 'user4',
+        email: 'iroh@example.com',
+        givenName: 'Iroh',
+        familyName: 'Coldstart',
```

Comment:
Do we have a login on the review app for this user?

Reply:
I think that will require an addition to add_cypress_test_users.ts

Reply:
Not as far as I know.  @mojotalantikite @macrael how do we add new review app users?

Reply:
done